app/main.py

```diff
@@ -1,6 +1,25 @@
+import os
+
+import requests
+
+# API_KEY = "b0480265525c4b09844183826222811"
+BASE_URL = "https://api.weatherapi.com/v1/current.json?key="
+API_KEY = os.environ.get("API_KEY")
+CITY = "Paris"
+
+URL = BASE_URL + API_KEY + "&q=" + CITY
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(URL).json()
+    print(
+        f"Country: {response['location']['country']}.\n"
+        f"City: {response['location']['name']}.\n"
+        f"Time zone: {response['location']['tz_id']}.\n"
+        f"Current time: {response['location']['localtime']}.\n"
+        f"Temperature: {response['current']['temp_c']} °C.\n"
+        f"Wind speed: {response['current']['wind_mph']} m/h."
```

Comment:
I don't know if it is required to make output similar to that which is in Readme file, so it is up to you. You can skip this comment and I'll approve it anyway